cal_ratio_trainer/build/build_main.py

```diff
@@ -7,6 +7,106 @@
 import dask.dataframe as dd
 import dask
 
+# Building the list of column names
+# Naively I want to do this just once so I don't need to reference it every time
+# I read in a new file?
+col_cluster_names_raw = [
+    "clus_pt",
+    "clus_eta",
+    "clus_phi",
+    "clus_l1hcal",
+    "clus_l1ecal",
+    "clus_l2hcal",
+    "clus_l2ecal",
+    "clus_l3hcal",
+    "clus_l3ecal",
+    "clus_l4ecal",
+    "clus_l4hcal",
+    "clus_time",
+]
+
+
+def _generate_col_cluster_names(index: int) -> List[str]:
+    "Generate the column names by appending a _{index} to each column name"
+    return [f"{name}_{index}" for name in col_cluster_names_raw]
+
+
+col_track_names_raw = [
+    "track_pt",
```

Comment:
Aren't these names already in `column_names.py`? We should have them only in a single place, using imports to share them.